apps/meeting/src/providers/VideoTileGridProvider/Utils.ts

```diff
@@ -96,8 +101,13 @@ export const calculateVideoSourcesToBeRendered = (
 export const updateDownlinkPreferences = (
   gridState: GridState,
   videoSourceState: VideoSourceState,
-  attendeeStates: { [attendeeId: string]: AttendeeState }
+  attendeeStates: { [attendeeId: string]: AttendeeState },
+  priorityBasedPolicy: VideoPriorityBasedPolicy | undefined
```

Comment:
I do not have a solution but is there a better place for this policy injection? Could you explain why do we need the `priorityBasedPolicy` in `VideoTileGridProvider`? May be @xuesichao can help here :)

Reply:
I don't think so, because this `Utils.ts` is actually a static file. And `updateDownlinkPreferences` method need this `VideoPriorityBasedPolicy`.

We may find some better solution, but that needs some refactoring.

Reply:
Well basically like Sichao said Util.js is not a component and this is needed so this is the only way unless you want me to rewrite this whole logic between video tile grid provider and state reducer :D

Reply:
Yeah, can fix later, no worries.